demo-extended/src/main/java/com/novoda/downloadmanager/demo/extended/DemoApplication.java

```diff
@@ -18,6 +20,9 @@ public void onCreate() {
         super.onCreate();
         notificationCustomiser = new DemoNotificationCustomiser(DemoApplication.this);
         oneRuleToBindThem = new OneRuleToBindThem();
+
+        DownloadManager downloadManager = DownloadManagerBuilder.from(this).build();
+        downloadManager.sanityCheckBatchStatuses();
```

Comment:
Not `restart` because this does not restart paused by the app downloads or other valid cases, this fixes invalid states of the downloads.

`onCreateApplication` yeah maybe as this should be called at that moment